### PR TITLE
(feat) : renamed billable services to chargeable

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
Update the title and description of billable services and commodities to `chargeable` to conform with industry standards

## Screenshots
Screenshot 2024-10-02 at 16 24 31

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by ojwanganto** [comment]
@donaldkibet - i think I only referred to one place where we could use charge item. I may have wrongly relied on FHIR for the change request and I would request that we liaise with BAs before merging this in

File: packages/esm-billing-app/src/billable-services/billables/charge-summary-table.component.tsx [hunk 1]
```diff
@@ -106,9 +107,9 @@ const ChargeSummaryTable: React.FC = () => {
   if (!chargeSummaryItems.length) {
     return (
       <EmptyState
-        headerTitle={t('clinicalCharges', 'Medical Invoice Items')}
+        headerTitle={t('chargeableServices', 'Chargeable Summary')}
```
Comment (ojwanganto) [inline packages/esm-billing-app/src/billable-services/billables/charge-summary-table.component.tsx:110]: It should be Charge items

File: packages/esm-billing-app/src/billable-services/billables/charge-summary-table.component.tsx [hunk 2]
```diff
@@ -119,8 +120,7 @@ const ChargeSummaryTable: React.FC = () => {
         {({ rows, headers, getHeaderProps, getRowProps, getTableProps, getToolbarProps, getTableContainerProps }) => (
           <TableContainer
             className={styles.tableContainer}
-            title={t('clinicalCharges', 'Medical Invoice Items')}
-            description={t('clinicalChargesDescription', 'Billable services and consumable prices')}
+            title={t('chargeableServices', 'Chargeable Summary')}
```
Comment (ojwanganto) [inline packages/esm-billing-app/src/billable-services/billables/charge-summary-table.component.tsx:123]: should be charge items summary

**Comment by ojwanganto** [comment]
Thanks, @donaldkibet. Let's just have the changes to reflect 'Charge items'. Chargeable has a probability connotation :)

**Comment by donaldkibet** [comment]
@ojwanganto 
https://github.com/user-attachments/assets/342883b1-e6d5-4923-b818-665b5ef18927

**Comment by donaldkibet** [comment]
Updated with the changes.

**Pull request merged**